### PR TITLE
Add bentoml task

### PR DESCRIPTION
**Pull request by iancoffey**
# Changes

Add a bentoml catalog task to allow for interesting CI patterns :)

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [ ] Yaml file complies with [yamllint](https://github.com/adrienverge/yamllint) rules.

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by iancoffey** [comment]
This is effectively blocked on https://github.com/bentoml/BentoML/pull/810 in that the usefulness of this command depends on it.

**Comment by vinamra28** [comment]
@iancoffey Thanks for adding this task :+1: 
Could we add some tests/examples for the task as how we can use it?

**Comment by iancoffey** [comment]
Ive added a test run and improved the read me wording. Ive also tested this out and it works as expected 🥳 

Since bentoml/BentoML#810 is merged this is ready for review, I will figure out the CI/CLA issues.

**Comment by vdemeester** [comment]
@iancoffey switched the base branch to master as it got merged with `v1beta1` now :angel:

**Comment by iancoffey** [comment]
I am not sure why CI is failing on this one 🤔

**Comment by dibyom** [comment]
hi @iancoffey The CI issue should be now fixed. If you rebase and push, the tests should pass!

**Comment by iancoffey** [comment]
Thanks for the CI help all. Looks like this is green now, and needs a review.

I have this running in a dev Pipeline now with Kaniko and it is working well.

cc @vinamra28 @dibyom

File: bentoml/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# BentoML
+
+This task allows operations on BentoML services.
```
Comment (ImJasonH) [inline bentoml/README.md:3]: Can you add a link to more info? I'm not familiar with this service.

Comment (iancoffey) [reply bentoml/README.md]: Ah, good call - I added a link to the projects repository.

Comment (iancoffey) [reply bentoml/README.md]: Which removed your lgtm, oops.

**Pull request merged**